src/main/java/com/my/FoodTruckBackend/customer/NewCustomerRequestBody.java

```diff
@@ -0,0 +1,28 @@
+package com.my.FoodTruckBackend.customer;
+
+import javax.validation.constraints.Email;
+import javax.validation.constraints.NotBlank;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Data
+public class NewCustomerRequestBody {
+    @NotBlank(message = "Please enter your first name")
+    private String firstName;
+
+    @NotBlank(message = "Please enter your last name")
+    private String lastName;
+
+    @Email(message = "Invalid email, please enter valid email address")
+    @NotBlank(message = "Please enter a valid email address")
+    private String email;
+
+    @NotBlank(message = "Please choose a unique password")
+    private String password;
+
+    @NotBlank(message = "Please re-enter your unique password")
+    private String confirmPassword;
```

Comment:
1. Should we add a max length for all of these fields? (according to table)
2. Should password have a min-length?

Reply:
It also looks like you allow empty spaces before/after the fields

Reply:
These constraints have been added